tools/runscript/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/TheThingsNetwork/lorawan-devices/tools/runscript
 
-go 1.18
+go 1.23.0
+
+toolchain go1.24.2
```

Comment:
Please revert changes in `go.mod` and `go.sum`. We'll do dependency updates outside of submissions to the contents.